docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Set asynchronous communication between Functions
+---
+
+This tutorial shows how you can connect two Functions using asynchronous communication. You can do it thanks to the In-cluster Eventing embedded in Kyma. The tutorial is based on Function from [Incluster Eventing example](https://github.com/kyma-project/examples/pull/188).
+
```

Comment:
We should add scenario description:
I.e: 

In this example we will demonstrate how to connect two function asynchronously.

We can think of very simple scenario of two functions, where the first function will be accepting incoming traffic via HTTP, sanitising the payload and publishing the content as an in-cluster event via kyma eventing.
Another function will be message receiver. It will subscribe to the given event type and will store the payload.